packages/react-components/src/components/OptionsButton.tsx

```diff
@@ -136,7 +139,7 @@ const generateDefaultMenuProps = (
           key: camera.id,
           text: camera.name,
           title: camera.name,
-          iconProps: { iconName: 'Camera' },
+          onRenderIcon: () => <Video20Regular style={{ ...iconStyle, ...{ color: theme.palette.themePrimary } }} />,
```

Comment:
Instead of setting the color manually in the style here, you should be able to use `primaryFill="currentColor"` as a prop on `Video20Regular` to ensure we don't lose any hover/active/disabled coloring (which may not apply but best to avoid overwriting the color when necessary, hopefully we'll be able to remove setting primary fill when https://github.com/microsoft/fluentui/issues/17804 is fixed).

Reply:
Unfortunately, it seems like using prop `primaryFill="currentColor` sets the color to `theme.palette.black`.
![image](https://user-images.githubusercontent.com/79475487/128139223-3191a472-8707-469d-a83e-66cf9572ea39.png)
![image](https://user-images.githubusercontent.com/79475487/128139243-230452fd-2bff-4c41-bedc-8603e0f82bc7.png)

Reply:
Ah right because we want the icon to be different from the text - my bad!